packages/components/src/common/CloseButton/CloseButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Types from 'prop-types';
+import classNames from 'classnames';
+import { Cross as CrossIcon } from '@transferwise/icons';
+
+export const CloseButton = ({ onClick, className }) => (
+  <button
+    className={classNames(
+      'tw-flow-navigation__close-button',
+      'btn-link',
+      'text-no-decoration',
+      className,
+    )}
+    aria-label="close button"
```

Comment:
I guess just _close_ would suffice? It'd be great if it was localised, but I understand adding i18n is out of scope for this PR, of course.
```suggestion
    aria-label="Close"
```